### ISSUE TITLE
Add headings to collection description box

### ISSUE THREAD

**Issue by JessWaggs** [UCSCLibrary/dams_project_mgmt#450]
### Descriptive Summary

Include bolded headings to content sections in the collection description box

![Screen Shot 2021-02-22 at 10 26 45 AM](https://user-images.githubusercontent.com/6412857/108778930-fed67880-751a-11eb-9551-4384f2f02f6e.png)

### Background 

Providing section headings to long blocks of text is helpful for usability.

### Acceptance Criteria

This is what done looks like

* [ ] The "Collection Contents" header appears above content information
* [ ] The "Harmful Language Statement" header appears, if such a statement is present (this type of statement is currently present on the Ruth-Marion Baruch collection and SC County Historic Photographs

**Comment by summer-cook** [comment]
Hi @rmjaffe - is it okay if I split this one into 2 tickets? The first part is super easy to accomplish and can be done today. The second part is a bit more tricky because there is an issue with the ScoobySnacks gem that it depends upon in order to complete.

**Comment by rmjaffe** [comment]
@summer-cook:  Absolutely.  You're welcome to carve things up how they make best sense to you as the person who's working on these things.  Let me know if/how I can help.  Also, let me or others on the UCSC team know if a ticket doesn't make sense, more information is wanting, or if the scope is too large and it should be broken into multiple tickets.  Thanks for asking, I hope the ScoobySnacks piece isn't too onerous!

**Comment by summer-cook** [comment]
Thank you!

**Comment by rschwab** [comment]
@crisr15 Same: was there a pull request for this? Wondering what the status is, we don't see any PR or MR for staging on this one.
